src/app/state/accountHistorySlice.ts

```diff
@@ -212,4 +259,24 @@ export const selectOrderHistory = createSelector(
   (orderHistory) => orderHistory.filter((order) => order.status !== "PENDING")
 );
 
+export const selectCurrentPairAddress = (state: RootState) =>
+  state.pairSelector.address;
+
+export const selectCombinedOrderHistory = createSelector(
+  (state: RootState) => state.accountHistory.orderHistory,
+  (state: RootState) => state.accountHistory.orderHistoryAllPairs,
+  (state: RootState) => state.accountHistory.hideOtherPairs,
+  (state: RootState) => selectCurrentPairAddress(state),
```

Comment:
`selectCurrentPairAddress(state)` seems to be not used, maybe remove if not needed.